dxr/templates/folder.html

```diff
@@ -27,14 +27,20 @@
     <thead>
       <tr>
         <th scope="col">Name</th>
+        {% for header in plugin_headers %}
+          <th scope="col">{{ header }}</th>
+        {% endfor %}
         <th scope="col">Modified</th>
```

Comment:
right, and just as note to self, https://bugzilla.mozilla.org/show_bug.cgi?id=1189576